src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Queries/GetTransactionsDownloadTests/WhenIGetTransactionsDownload.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Application.Formatters.TransactionDowloads;
+using SFA.DAS.EAS.Application.Queries.GetTransactionsDownloadResultViewModel;
+using SFA.DAS.EAS.Domain.Data.Repositories;
+using SFA.DAS.EAS.Domain.Models.AccountTeam;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Domain.Models.UserProfile;
+using SFA.DAS.HashingService;
+
+namespace SFA.DAS.EAS.Application.UnitTests.Queries.GetTransactionsDownloadTests
+{
+    public class WhenIGetTransactionsDownload
+    {
+        private const string HashedAccountId = "ABC123";
+        private const long UserId = 222222;
+        private const long FailingUserId = 333333;
+
+
+        private readonly TransactionsDownloadStartDateMonthYearDateTime _fromDate =
+            new TransactionsDownloadStartDateMonthYearDateTime
+            {
+                Month = 1,
+                Year = 2000,
+            };
+
+        private readonly TransactionsDownloadEndDateMonthYearDateTime _toDate =
+            new TransactionsDownloadEndDateMonthYearDateTime
+            {
+                Month = 1,
+                Year = 2000,
+            };
+
+        private readonly TransactionsDownloadStartDateMonthYearDateTime _fromDateNoResults =
+            new TransactionsDownloadStartDateMonthYearDateTime
+            {
+                Month = 1,
+                Year = 2000,
+            };
+
+        private readonly TransactionsDownloadEndDateMonthYearDateTime _toDateNoResults =
+            new TransactionsDownloadEndDateMonthYearDateTime
+            {
+                Month = 1,
+                Year = 2000,
+            };
+
+        private GetTransactionsDownloadQueryHandler _handler;
+        private GetTransactionsDownloadQuery _query;
+        private GetTransactionsDownloadResponse _response;
```

Comment:
I think there's a unit test missing that asserts against the response being returned?

Reply:
Actioned